tests/postgres_tests/test_aggregates.py

```diff
@@ -128,6 +128,24 @@ def test_json_agg_empty(self):
         self.assertEqual(values, json.loads('{"jsonagg": []}'))
 
 
+class TestArrayAggDistinct(PostgreSQLTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        AggregateTestModel.objects.create(char_field='Foo')
+        AggregateTestModel.objects.create(char_field='Foo')
+        AggregateTestModel.objects.create(char_field='Bar')
+
+    def test_string_agg_distinct_false(self):
```

Comment:
I think the `TestStringAggregateDistinct` class could be reused considering the setup methods are the same -- just remove `String` from the class name.